Python/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:latest
```

Comment:
Make use of `ENV `variables and use it in the code
Try to also use command line args

Reply:
IMPLEMENTED BOTH